docs(gen): tutorials requeriments update part ii (#2924)
scaleway/docs-content: tutorials/deploy-wordpress-managed-database-mysql8/index.mdx

> 4. The installation of your blog has been successfully. You can log in now and write your first articles or configure the look and feel of WordPress by installing additional themes and modules.

Reviewer: ```suggestion
4. You can now log in to your blog and write your first articles, or configure the look and feel by installing additional themes and modules.
```
